frontend/javascripts/oxalis/model/accessors/dataset_layer_transformation_accessor.ts

```diff
@@ -443,6 +460,28 @@ function _doAllLayersHaveTheSameRotation(dataLayers: Array<APIDataLayer>): boole
 
 export const doAllLayersHaveTheSameRotation = _.memoize(_doAllLayersHaveTheSameRotation);
 
+export function transformationEqualsAffineIdentityTransform(
+  transformations: CoordinateTransformation[],
+): boolean {
+  const hasValidTransformationCount = transformations.length;
```

Comment:
```suggestion
  const hasValidTransformationCount = transformations.length > 0;
```

reads a bit nicer imo